Snowdog_Components/components/Molecules/table/_table.scss

```diff
@@ -84,6 +84,25 @@
         }
     }
 
+    &--tfoot-border {
+        tfoot {
+            tr {
+                th {
+                    border-top: $table__tfoot-th-border-top\@medium;
+                    border-bottom: 0;
+
+                    &[scope="row"] {
+                        border-right-width: 1px;
+                    }
+                }
+                td {
+                    border-top: $table__tfoot-td-border-top\@medium;
```

Comment:
same as above
`\@medium` means that the style is applied for `@include mq($screen-m)` but I can't see any media queries here so it should be removed. one thing worth checking first is whether the styles look good on mobile. can you confirm?